### DISCUSSION TITLE
SkiaSharp version with vulnerabilities

### DISCUSSION THREAD

**Discussion by fredericoregateiro** [AvaloniaUI/Avalonia#15924] [Q&A] [answered]
Some versions have vulnerabilities, and the avalonia installs the 2.88.3 that have this problem.
Are there any plans to upgrade the skiasharp version for avalonia v11.x?

![image](https://github.com/AvaloniaUI/Avalonia/assets/6304701/ca9e03c8-c550-43cb-8d13-cc8c56a5769f)

**Comment by MikeCodesDotNET** [comment]
Hi @fredericoregateiro, 

If you look at our [dependencies](https://www.nuget.org/packages/Avalonia.Skia/11.0.10#dependencies-body-tab), you will notice that we depend on 2.88.7, which has no known vulnerabilities.

**Reply by fredericoregateiro** [reply, marked as answer]
Thanks @MikeCodesDotNET 

I've been looking a bit more in the dependencies and it looks like the skia dependency is not from the avalonia project but from the FluentAvaloniaUI.
